react/Title.tsx

```diff
@@ -18,7 +18,39 @@ const messages = defineMessages({
   },
 })
 
-const CartTitle: StorefrontFunctionComponent<Props> = ({ title }) => {
+const countCartItems = (countMode: MinicartTotalItemsType, arr: Item[]) => {
+  if (countMode === 'distinctAvailable') {
+    return arr.reduce((itemQuantity: number, item: Item) => {
+      if (item.availability === 'available') {
+        return itemQuantity + 1
+      }
+      return itemQuantity
+    }, 0)
```

Comment:
suggestion (non-blocking): I think this operation could be simplified with the following. What do you think? (obs: untested)
```suggestion
	return arr.filter(item => item.availability === 'available').length
```

Reply:
Totally agreed. I copied this method from the minicart component to make sure cart and minicart was using the same count method. I'll test this code and commit it. :D